src/Framework/ref/Microsoft.AspNetCore.App.Ref.Internal.csproj

```diff
@@ -0,0 +1,14 @@
+<Project>
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
```

Comment:
Why not `<Project Sdk="Microsoft.NET.Sdk">`❔

Reply:
I just trimmed down the `app.ref` proj file to create this one, and that file imports the sdk.props/targets files directly. I'll change this one to do it in the project header.